src/view/show_mode/editor/show_ui_widgets/cue_control.py

```diff
@@ -71,21 +93,41 @@ def construct_widget(self, parent: QWidget | None, enabled: bool):
             item.annotated_data = cue
             cue_list.addItem(item)
         cue_list.setEnabled(enabled)
-        cue_list.setMinimumHeight(100)
+        cue_list.setMinimumHeight(300)
         if enabled:
             self._player_cue_list_widget = cue_list
         else:
             self._config_cue_list_widget = cue_list
         layout.addWidget(cue_list)
+
+        self._progressBar.setParent(w)
+        self._progressBar.setEnabled(enabled)
+        self._progressBar.setMinimumHeight(60)
+        self._progressBar.setMinimum(0)
+        self._progressBar.setMaximum(20)
+        layout.addWidget(self._progressBar)
```

Comment:
do you think, we dont need anything like a searchbar, or is there anything similar?

Reply:
If you use `QLineEdit` with a `QCompleter` you basically have a search bar. You mean to search for the next cue, right? That could be useful.

Reply:
not for the next cue (right now i dont have access to that data). we could implement something to start at specific time with fish, i thought just for having a visual feedback of the actual cue

Reply:
What do you need a search bar for then? Regarding the progress of the current cue I would simply update a text label displayed within the show ui widget.

Reply:
i just thought as visual benefit.